Libplanet/Net/AppProtocolVersion.cs

```diff
@@ -66,6 +66,38 @@ Address signer
             Signer = signer;
         }
 
+        /// <summary>
+        /// A token string which serializes an <see cref="AppProtocolVersion"/>.
+        /// <para>As this is designed to be easy to copy and paste, the format consists of only
+        /// printable characters in the ASCII character set.</para>
+        /// <para>A token can be deserialized into an <see cref="AppProtocolVersion"/> through
+        /// <see cref="FromToken(string)"/> method.</para>
+        /// </summary>
+        /// <seealso cref="FromToken(string)"/>
+        [Pure]
+        public string Token
+        {
+            get
+            {
+                string sig = Convert.ToBase64String(
+                    Signature.ToBuilder().ToArray(),
+                    Base64FormattingOptions.None
+                ).Replace('/', '.');
```

Comment:
How about using `;` or `.` as delimiter instead of `/`? then it seems can be omitted.